application/models/Keyword_model.php

```diff
@@ -9,6 +9,143 @@ public function get_applied($list_keywords)
 
 
     }
+    
+ /**   
```

Comment:
We can just drop this, especially if we're dropping the migration as stated above and relying on the cron job.